core/runtime/src/main/java/io/quarkiverse/jberet/runtime/QuarkusJobOperator.java

```diff
@@ -33,17 +34,20 @@
     private final BatchEnvironment batchEnvironment;
     private final Map<String, Job> jobs;
     private final JBeretConfig config;
+    private final ThreadPoolConfig threadPoolConfig;
 
     public QuarkusJobOperator(
             final JBeretConfig config,
             final ManagedExecutor managedExecutor,
             final TransactionManager transactionManager,
-            final List<Job> jobs) {
+            final List<Job> jobs,
+            final ThreadPoolConfig threadPoolConfig) {
```

Comment:
Can you group both config classes? So, arguments order should be first the config objects and then the rest?

Reply:
@radcortez , Order of config objects is done. Is your idea to group both config files to one file? But as Config root for both config files are different it is best not to merge. If your idea is different then that, please guide me through what approach you have.

Reply:
Thanks. It was just the order :)